daemons/pacemakerd/pacemakerd.c

```diff
@@ -416,15 +46,6 @@ pcmk_sigquit(int nsig)
     pcmk__panic(__func__);
 }
 
-void
-pcmk_shutdown(int nsig)
-{
-    if (shutdown_trigger == NULL) {
-        shutdown_trigger = mainloop_add_trigger(G_PRIORITY_HIGH, pcmk_shutdown_worker, NULL);
-    }
-    mainloop_set_trigger(shutdown_trigger);
-}
```

Comment:
pcmk_shutdown() is debatable too (along with startup_trigger/shutdown_trigger) since it's used by the main pacemakerd process itself to handle SIGTERM/SIGINT (i.e. the flow of the main process), but we can leave that for later to figure out a good split